docs/jobs.md

```diff
@@ -0,0 +1,432 @@
+Cozy Jobs
+=========
+
+Jobs are designed to represent asynchronous tasks that your cozy can execute. These tasks can be scheduled in advance, recurring or suddenn and provide various services.
+
+At the time, we do not provide "generic" and programmable tasks. This list of available workers is part of the defined API.
+
+The job queue can be either local the cozy-stack when used as a monolitic instance (self-hosted for instance) or distributed via a redis server for distributed infrastructures.
+
+This doc introduces two cozy types:
+
+  - `io.cozy.jobs` for jobs
+  - `io.cozy.launchers` for launchers
+
+
+Launchers
+---------
+
+Jobs can be launched by three types of launchers:
+
+  - `@cron` to schedule recurring jobs, either periodic or scheduled at specific times
```

Comment:
Is it possible to schedule a one-time job? Like send me a notification for this event in my calendar 10 minutes before it start. I don't see how to do that with the `@cron` syntax below.